src/controllers/request-controller.ts

```diff
@@ -113,7 +113,7 @@ export class RequestController {
       }
     } catch (err) {
       const errmsg = `Creating request with streamId ${req.body.streamId} and commit CID ${req.body.cid} failed: ${err.message}`
-      logger.err(errmsg)
+      logger.err(err)
```

Comment:
this change loses information (the streamid and commit CID that originated the error).
I put up https://github.com/ceramicnetwork/ceramic-anchor-service/pull/776 to log both the original message and also the stack the trace

Reply:
Ah good, point. Ty 🙏🏼